src/main/java/frc/robot/constants/ScoringConstants.java

```diff
@@ -52,6 +52,8 @@ public class ScoringConstants {
     public static final double aimAngleMarginRotations = Units.degreesToRotations(1);
     public static final double aimAngleVelocityMargin = 2.0;
 
+    public static final double minDistanceAlignmentNeeded = 1.3; // TODO: Tune this value
```

Comment:
Rather than using a distance (that we aren't likely computing reliably anyway) let's add a button somewhere as a human input that we want to turn off drive alignment while held

Reply:
My thought was that the driver would press localize when against the subwoofer and then it would immediately realize it was close enough and shoot. Which button would you suggest using for human input? (I think left bumper may be free?)

Reply:
We would be forced to only do center subwoofer if we rely on the relocalization button. If left bumper is free that sounds fine to me